eslint/babel-eslint-parser/package.json

```diff
@@ -52,23 +48,8 @@
     "@babel/preset-env": "^7.1.5",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
-    "babel-eslint": "^10.0.2",
     "dedent": "^0.7.0",
-    "eslint": "^6.0.1",
```

Comment:
I don't think we may remove eslint here because `eslint/babel-eslint-parser/test/specs/integration.js` depends on `eslint`.